server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -98,7 +99,7 @@ public class ShuffleServer {
   private volatile boolean running;
   private ExecutorService executorService;
   private Future<?> decommissionFuture;
-  private boolean nettyServerEnabled;
+  private boolean isNettyServerEnabled;
```

Comment:
we should use `nettyServerEnabled` to keep consistent with the style of other places.

Reply:
> we should use `nettyServerEnabled` to keep consistent with the style of other places.

Done. I also found places like `isMetricsEnabled` and `isMemoryShuffleEnabled`

Reply:
We can change them to unify this style.

Reply:
> We can change them to unify this style.

Do we change this in the current PR?

Reply:
No need.